client/src/theme/styles/components/frontend/project-collection/_header.scss

```diff
@@ -105,32 +94,26 @@ $_font-family: 'sofia-pro', 'trueno', sans-serif;
     > .manicon-svg {
       position: relative;
       flex-shrink: 0;
-      width: $_icon-size;
-      height: $_icon-size;
       color: #b3b3b3;
     }
   }
 
   &__hero-image {
-    flex-shrink: 0;
-    min-width: 100%;
-    min-height: 100%;
+    width: 100%;
+
+    &--square {
+      width: auto;
+      height: 100%;
+    }
```

Comment:
This is still overflowing even though the image has the correct aspect ratio.

<img width="575" alt="Screen Shot 2020-07-23 at 10 27 36 AM" src="https://user-images.githubusercontent.com/12454203/88318328-6b606f00-cccf-11ea-926c-07757948ed54.png">

I think the image wrapper approach is overcomplicating things here. You could omit the wrapper entirely and just set a fixed pixel size for the image and use `object-fit: cover` to make sure the image isn't stretched.

```scss
img {
  width: 340px;
  height: 340px;
  object-fit: cover;
}
```

The same approach should work for all three layouts.

Reply:
Huh. I did not see the image overflow in any of my tests after my most recent changes, but I went ahead with the changes you suggested and they work very well.